src/replication/transfer/image/transfer.go

```diff
@@ -243,12 +245,13 @@ func (t *transfer) copyContent(content distribution.Descriptor, srcRepo, dstRepo
 	// the media type of the layer or config can be "application/octet-stream",
 	// schema1.MediaTypeManifestLayer, schema2.MediaTypeLayer, schema2.MediaTypeImageConfig
 	default:
-		return t.copyBlob(srcRepo, dstRepo, digest)
+		return t.copyBlob(srcRepo, dstRepo, digest, size)
```

Comment:
`t.copyBlob(srcRepo, dstRepo, digest, size)` to `t.copyBlob(srcRepo, dstRepo, digest, conten.size)`